compass/landice/tests/ismip6_run_ais/run_model.py

```diff
@@ -0,0 +1,156 @@
+import os
+from compass.model import run_model, make_graph_file
+from compass.step import Step
+
+
+class RunModel(Step):
+    """
+    A step for performing forward MALI runs as part of Antarctica test cases.
+
+    Attributes
+    ----------
+    mesh_file : str
+        The name of the mesh file being used
+
+    mesh_type : str
+        The resolution or mesh type of the test case
+    """
+
+    def __init__(self, test_case, mesh_type, name='run_model'):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        mesh_file : str
+            The name of the mesh file being used
+
+        mesh_type : {'high', 'mid'}
+            The resolution or mesh type of the test case
+        """
+
+        self.mesh_file = None
+        self.mesh_type = mesh_type
+
+        super().__init__(test_case=test_case, name=name)
+
+    def setup(self, velo_solver="FO"):
```

Comment:
Also, this gets called by the framework without using any arguments:
https://github.com/MPAS-Dev/compass/blob/ff4e402533bf762d06a7b847eec290df232a44d9/compass/setup.py#L289
so any parameters you add here would always take the default value.  If you wanted to have various `velo_solver` values, this would need to happen in the constructor (`__init__()`) rather than here.